### DISCUSSION TITLE
Multiple:true option is not working in OnePlus Android (Version: 14),bug

### DISCUSSION THREAD

**Discussion by venna-kalyani** [capawesome-team/capacitor-plugins#165] [General]
Hi @robingenz  , 
We are using capacitor 4 and file picker of 0.6.3 versions in our application.
Due to some limitations we cannot upgrade capacitor version to 5 at this movemet.

With the above combination , we are encountering multiple selection issue issue.
**multiple:true option is not working in OnePlus Android 14 version device.**
But Its working as expected in Google pixel and Samsung devices.

**Code snippet**
`const pickMedia = async () => {
  const result = await FilePicker.pickMedia({
    multiple: true,
  });
};`

`const pickFiles = async () => {const result = await FilePicker.pickFiles({types: ['image/png'],multiple: true,});};`

Expected :
Should be able to select multiple photos/documents in OnePlus android 14 version mobile device as well.

Please help us with this issue how can we achieve multiple selection.

**Comment by robingenz** [comment]
Unfortunately, we can only offer support for the latest versions. You should update to Capacitor 6.

**Reply by venna-kalyani** [reply]
Thanks for the quick replay @robingenz

**Comment by venna-kalyani** [comment]
@robingenz , I am trying to create a blob with the following code and getting a response as an empty object {}.
` const webPath = Capacitor.convertFileSrc(file?.path);
        console.log('webPath::::::', webPath) // get log webPath:::::: capacitor://localhost:8080/_capacitor_file_/var/mobile/Containers/Data/Application/82418282-5158-4E2A-860E-C44B17895FAA/Library/Caches/IMG_5772.png
        const response = await fetch(webPath);
        console.log('response=====>', response) // getting empty response {}
        let blob = await response?.blob();
        console.log('blob::::', blob); // getting empty blob {}`


Does these methods (convertFileSrc and fetch) required any additional level permissions from native/core Android and iOS files. As the webpath is showing as //localhost:8080/ on the mobile, do we need to modify anything in info.plist / AndroidManifest.xml files? to access them.

If not what could be the reason for empty response though there is a valid webpath.

Could you please help me with the solution.

**Reply by venna-kalyani** [reply]
@robingenz  still we are facing the same issue. could you please help us .?

**Reply by robingenz** [reply]
That are not enough information to help you. This issue is related to the Capacitor Core. The File Picker plugin is only responsible for providing a valid path.
